src/backend/distributed/executor/citus_custom_scan.c

```diff
@@ -162,6 +162,11 @@ CitusBeginScan(CustomScanState *node, EState *estate, int eflags)
 	MarkCitusInitiatedCoordinatorBackend();
 
 	scanState = (CitusScanState *) node;
+
+#if PG_VERSION_NUM >= 120000
```

Comment:
This is necessary because pg12 gives us a `TTSOpsVirtual` which doesn't use with tuplestore